libcontainer/criu_linux.go

```diff
@@ -1151,6 +1151,9 @@ func (c *Container) criuNotifications(resp *criurpc.CriuResp, process *Process,
 		}
 		// create a timestamp indicating when the restored checkpoint was started
 		c.created = time.Now().UTC()
+		if !c.config.Namespaces.Contains(configs.NEWTIME) && configs.IsNamespaceSupported(configs.NEWTIME) {
```

Comment:
Do we also need to check criu version here (since criu < 3.14 does not add timens)?

Reply:
The alternative is just require criu >= v3.14 (or v3.15) globally (from `Checkpoint` and `Restore`), as it's been quite a while since it was released.